code/datums/traits/negative.dm

```diff
@@ -120,13 +120,13 @@
 
 /datum/quirk/foreigner/add()
 	var/mob/living/carbon/human/H = quirk_holder
-	if(ishuman(H) && H.job != JOB_CURATOR)
+	if(ishuman(H) && H.job != "Curator")
```

Comment:
why are you changing this back to the string when you already haved defines for those strings in your branch

Reply:
because I am going to make it at another PR that would make large change about job string defines